gems/sorbet-runtime/sorbet-runtime.gemspec

```diff
@@ -18,4 +18,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake'
   # for reproducing race conditions in tests
   s.add_development_dependency 'concurrent-ruby', '~> 1.1.5'
+  s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry-byebug'
```

Comment:
I found these useful for debugging errors here and suspect it would be similarly useful for others.